bb.edn

```diff
@@ -0,0 +1,18 @@
+{:paths ["resources"]
+
+ :tasks
+ {:requires [[babashka.deps :as deps]
+             [babashka.process :as p]]
+
+  gen-script {:doc "Regen `./deps[.clj|.bat]` from `src/borkdude/deps.clj`."
+              :task (let [script "script/gen_symclj"
+                          bb (str \" (.get (.command (.info (java.lang.ProcessHandle/current)))) \")]
```

Comment:
You could just use `load-file` here I think

Reply:
Ah yes of course, thanks for the tip! 

Done.